### ISSUE TITLE
Script error in steam_boot (sys_core)

### ISSUE THREAD

**Issue by Cuel** [IDI-Systems/acre2#76]
**Arma 3 Version:** Stable
**CBA Version:** 3.1.0.160928
**ACRE2 Version:** 2.3.0.913

**Mods:**
- `@CBA_A3`
- `@acre2`

**Description:**
Script error from steam_boot on launch. 
This is with a fresh (the addon older is called _fresh in rpt, same error without it) download of .913 and latest stable CBA

```
 0:36:40 Error in expression <nsion format["2VER%1", _version];
INFO_1("Steam Initialization Result: %1",_res)>
 0:36:40   Error position: <("Steam Initialization Result: %1",_res)>
 0:36:40   Error Missing ;
 0:36:40 File idi\acre\addons\sys_core\steam_boot.sqf, line 8
```

I've also tried moving the folders in the A3 root folder, same error

**Steps to reproduce:**
Launch the game

**Where did the issue occur?**
Local, Dedicated server (unix), other machines (players)

**RPT log file:**
http://pastebin.com/MuSNYxfi

**Comment by Cuel** [comment]
The file is missing an #include

**Comment by Cuel** [comment]
fixed by #77
